core/src/main/java/nl/nn/adapterframework/webcontrol/api/Base.java

```diff
@@ -76,8 +76,16 @@ public abstract class Base implements ApplicationContextAware {
 
 	public Response callGateway(RequestMessageBuilder input) throws ApiException {
 		Gateway gateway = getApplicationContext().getBean("gateway", Gateway.class);
-		Message<?> response = gateway.execute(input.build());
-		return BusMessageUtils.convertToJaxRsResponse(response);
+		Message<?> response = gateway.requestReply(input.build());
+		if(response != null) {
+			return BusMessageUtils.convertToJaxRsResponse(response);
+		}
+		return Response.noContent().build();
+	}
+	public Response callFFGateway(RequestMessageBuilder input) throws ApiException {
```

Comment:
Je hebt hem 'FF' genoemd, maar het is volgens mij alleen maar 'Asynchroon'. 
De 'forget' in de zin van 'gegarandeerde verwerking' is er niet, of is niet waar het om gaat.
Ik denk dat 'callAsynchronousGateway' de lading beter dekt.
En misschien moet de andere method dan 'callSynchronousGateway' gaan heten.